packages/tpetra/core/ext/TpetraExt_MatrixMatrix_ExtraKernels_def.hpp

```diff
@@ -289,7 +289,7 @@ void mult_A_B_newmatrix_LowThreadGustavsonKernel(CrsMatrixStruct<Scalar, LocalOr
   copy_out_from_thread_memory(thread_total_nnz,tl_colind,tl_values,m,thread_chunk,row_mapC,entriesC,valuesC);
 
 #ifdef HAVE_TPETRA_MMM_TIMINGS
-  MM.~(); // destruct the 'Core'
+  MM.~TimeMonitor(); // destruct the 'Core'
```

Comment:
This means that MM's destructor will get called twice -- once here, and once at the end of scope.  In this case, it means that the timer's `stop()` method might get called twice.